sql-plugin/src/main/scala/com/nvidia/spark/rapids/TypeChecks.scala

```diff
@@ -2193,15 +2211,119 @@ object SupportedOpsForTools {
     }
   }
 
-  def help(): Unit = {
-    outputSupportIO()
+  private def operatorMappingWithScore(): Unit = {
+    val header = Seq("CPUOperator", "Score")
+    println(header.mkString(","))
+    GpuOverrides.execs.values.toSeq.sortBy(_.tag.toString).foreach { rule =>
+      val checks = rule.getChecks
+      if (rule.isVisible && checks.forall(_.shown)) {
+        val cpuName = rule.tag.runtimeClass.getSimpleName
+        val allCols = Seq(cpuName, "2")
+        println(s"${allCols.map(replaceDelimiter(_, ",")).mkString(",")}")
+      }
+    }
+
+    GpuOverrides.expressions.values.toSeq.sortBy(_.tag.runtimeClass.getSimpleName).foreach { rule =>
+      val checks = rule.getChecks
+      if (rule.isVisible && checks.forall(_.shown)) {
+        val cpuName = rule.tag.runtimeClass.getSimpleName
+        val allCols = Seq(cpuName, "3")
+        println(s"${allCols.map(replaceDelimiter(_, ",")).mkString(",")}")
+      }
+    }
+  }
+
+  private def outputSupportedExecs(): Unit = {
+    // TODO Look at what we have for defaults for some configs because if the configs are off
```

Comment:
we should check the rule.disabledMsg.isDefined to see if they are disabled by default (CollectLimitExec is one example).  

We have other configs to control certain execs like ENABLE_LEFT_OUTER_JOIN, lets file a follow up to figure out a good way to account for those.

Reply:
Filed follow on issue - https://github.com/NVIDIA/spark-rapids/issues/5305